experimenter/experimenter/experiments/admin.py

```diff
@@ -326,16 +327,22 @@ class NimbusExperimentAdmin(
     filter_horizontal = ("excluded_experiments", "required_experiments")
 
 
-class NimbusFeatureConfigAdmin(NoDeleteAdminMixin, admin.ModelAdmin[NimbusFeatureConfig]):
+class NimbusFeatureConfigAdmin(ReadOnlyAdminMixin, admin.ModelAdmin[NimbusFeatureConfig]):
     prepopulated_fields = {"slug": ("name",)}
-    list_filter = ("application", "read_only")
-    list_display = ("name", "application", "read_only")
+    list_filter = ("application",)
+    list_display = ("name", "application")
     exclude = ("read_only",)
 
     def has_change_permission(self, request, obj: Optional[NimbusFeatureConfig] = None):
         return obj and not obj.read_only
 
 
+class NimbusVersionedSchemaAdmin(
+    ReadOnlyAdminMixin, admin.ModelAdmin[NimbusVersionedSchema]
+):
+    pass
```

Comment:
Oh we should add it as an inline into the feature config admin so we can still see it all in one place.